Makefile

```diff
@@ -19,7 +19,13 @@ run: all
 build-sass: $(SASS_SRC)
 	$(SASS_COMPILER) $(MAIN_FILE) $(OUTPUT_FILE)
 
+build-docker-image:
+	docker build -t 2016-site .
+
+docker:
```

Comment:
shouldn't `docker` depend on `build-docker-image` ? at least looks like `build-docker-image` should depend on the Dockerfile